app/src/main/java/com/bytemedrive/file/shared/control/FileListItemRepository.kt

```diff
@@ -18,6 +18,8 @@ class FileListItemRepository(
 
     suspend fun getAll(folderId: UUID? = null) = itemDao.getAll(folderId = folderId).map(::FileListItem)
 
+    fun getAllFlow(folderId: UUID? = null) = itemDao.getAllFlow(folderId = folderId).map { it.map(::FileListItem) }
```

Comment:
`itemDao.getAllFlow(folderId = folderId).map { it.map(::FileListItem) }.flowOn(ioDispatcher)`